Tools/Pipeline/Controls/PropertyCells/CellRefs.cs

```diff
@@ -3,27 +3,31 @@
 // file 'LICENSE.txt', which is part of this source code package.
 
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using Eto.Forms;
 
 namespace MonoGame.Tools.Pipeline
 {
     public class CellRefs : CellBase
     {
-        public CellRefs(string category, string name, object value, EventHandler eventHandler) : base(category, name, value, eventHandler)
+        public CellRefs(string category, string name, object value, EventHandler eventHandler)
+            : base(category, name, value, eventHandler)
         {
             if (Value == null)
                 Value = new List<string>();
 
-            DisplayValue = (Value as List<string>).Count > 0 ? "Collection" : "None";
+            DisplayValue = (Value as ICollection).Count > 0 ? "Collection" : "None";
```

Comment:
I can't actually manage to replicate the problem now as I've been changing a bunch of things with my project to get stuff working again, but at some point Value was being passed in as a Collection<SkinInfo> somehow, and not having a full understanding of the source I simply changed  it to support collections and it worked.